### DISCUSSION TITLE
How to remove the hyphens at the end of each line

### DISCUSSION THREAD

**Discussion by ul-shunei** [wojtekmaj/react-pdf#1342] [Q&A] [answered]
Long sentence is split by the default hyphenation callback, when it is rendered, each line has a `-` appended. How to remove these hyphens?

Sample
Original sentence: `The restrictions are meant to keep conversations from getting weird. `
PDF: 
```
The restrictions are meant-
to keep conversations from-
getting weird. 
```

**Comment by SunnySze0000** [comment]
#1679

**Comment by wojtekmaj** [comment, marked as answer]
It looks like you confused React-PDF with [@react-pdf/renderer](https://github.com/diegomura/react-pdf). Please file an issue in their repo instead.
